pkg/route/ingress/ingress_test.go

```diff
@@ -1094,6 +1094,84 @@ func TestController_sync(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "create route - custom passthrough ingresscontroller",
+			fields: fields{
+				i: &ingressLister{Items: []*networkingv1.Ingress{
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:        "1",
+							Namespace:   "test",
+							Annotations: map[string]string{"route.openshift.io/termination": "passthrough"},
+						},
+						Spec: networkingv1.IngressSpec{
+							IngressClassName: &openshiftCustomIngressClassName,
```

Comment:
Is there a reason you don't use `openshiftDefaultIngressClassName` here and omit `ic` below?